languages/woocommerce-related-accessories-de_DE.po

```diff
@@ -83,6 +55,27 @@ msgstr "Wähle eine Option"
 msgid "Reset accessories"
 msgstr "Zubehör ausblenden"
 
+#~ msgid "Pads and pillows"
```

Comment:
Could you also please run the `purge deleted translations` from the `Catalogue` menu in PoEdit to remove these?
 
![image](https://user-images.githubusercontent.com/863823/128736021-791d1d67-7ddf-4146-8d4a-4fe022742677.png)

Reply:
Thanks! Done.